bin/plugin/lib/milestone.js

```diff
@@ -14,18 +20,13 @@
  * @param {string} repo    Repository name.
  * @param {string} title   Milestone title.
  *
- * @return {Promise<OktokitIssuesListMilestonesForRepoResponseItem|void>} Promise resolving to milestone, if exists.
+ * @return {Promise<listMilestonesResponse["data"]|undefined>} Promise resolving to milestone, if exists.
```

Comment:
> There should only be a single value returned from this, hence an error about mismatched types on how `listMilestonesResponse["data"]` is an array.

Thanks! I got a bit caught up in the JSDoc syntax error and missed that this one was actually (also) a TS problem.

Thanks for your suggested fixes! I think I'll go with the `Milestone` (from `@octokit/openapi-types`, which I was unaware of), since it's the most semantic option.

> Or maybe it's not worth documenting it, and just letting the inferred type go through :shrug:

I seem to be getting `any` in that case though :thinking: (looking at the call site in `changelog.js`).